### PR TITLE
Planning: OpenSpace: fix trajectory partition

### PR DESCRIPTION
**Pull request by JasonZhou404**

### CODE REVIEW

File: modules/planning/tasks/optimizers/open_space_trajectory_partition/open_space_trajectory_partition.cc [hunk 1]
```diff
@@ -77,7 +77,7 @@ Status OpenSpaceTrajectoryPartition::Process() {
   TrajGearPair* current_trajectory = &(paritioned_trajectories_ptr->back());
   // Set initial gear position for first ADCTrajectory depending on v
   // and check potential edge cases
-  constexpr double kepsilon = 1e-8;
+  constexpr double kepsilon = 0.0;
```
Comment (xiaoxq) [inline modules/planning/tasks/optimizers/open_space_trajectory_partition/open_space_trajectory_partition.cc]: kEpsilon

File: modules/planning/tasks/optimizers/open_space_trajectory_partition/open_space_trajectory_partition.cc [hunk 2]
```diff
@@ -219,11 +239,19 @@ Status OpenSpaceTrajectoryPartition::Process() {
       vehicle_state.gear() == canbus::Chassis::GEAR_REVERSE
           ? NormalizeAngle(vehicle_state.heading() + M_PI)
           : vehicle_state.heading();
+  // TODO(Jinyun) move to confs
```
Comment (xiaoxq) [inline modules/planning/tasks/optimizers/open_space_trajectory_partition/open_space_trajectory_partition.cc]: `TODO(Jinyun): Move to confs.`

File: modules/planning/tasks/optimizers/open_space_trajectory_partition/open_space_trajectory_partition.cc (hunk 1 above)
Comment (JasonZhou404) [reply modules/planning/tasks/optimizers/open_space_trajectory_partition/open_space_trajectory_partition.cc]: fixed

File: modules/planning/tasks/optimizers/open_space_trajectory_partition/open_space_trajectory_partition.cc (hunk 2 above)
Comment (JasonZhou404) [reply modules/planning/tasks/optimizers/open_space_trajectory_partition/open_space_trajectory_partition.cc]: fixed

**Pull request merged**